Gopkg.toml

```diff
@@ -37,15 +37,15 @@
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.12.0-beta.1"
+  branch = "release-1.12"
 
 [[constraint]]
   name = "k8s.io/csi-api"
   branch = "release-1.12"
 
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.12.0-beta.1"
+  version = "kubernetes-1.12.0-rc.2"
```

Comment:
not sure if this or `kubernetes-1.12.1-beta.0` is more appropriate, your call

Reply:
I think 1.12.0-rc.2 is better because it limits client-go to stay below 1.12.1.